controllers/update_pod_config.go

```diff
@@ -66,10 +66,18 @@ func (u UpdatePodConfig) Reconcile(r *FoundationDBClusterReconciler, context ctx
 		if !synced {
 			allSynced = false
 			log.Info("Update dynamic Pod config", "namespace", cluster.Namespace, "cluster", cluster.Name, "processGroupID", instance.GetInstanceID(), "synced", synced, "error", err)
+
+			instance.Metadata.Annotations[fdbtypes.OutdatedConfigMapKey] = time.Now().Format(time.RFC3339)
+			err = r.PodLifecycleManager.UpdateMetadata(r, context, cluster, instance)
+			if err != nil {
+				allSynced = false
+				log.Info("Update Pod metadata", "namespace", configMap.Namespace, "cluster", cluster.Name, "processGroupID", instance.GetInstanceID(), "error", err)
+			}
 			continue
 		}
 
 		instance.Metadata.Annotations[fdbtypes.LastConfigMapKey] = configMapHash
+		delete(instance.Metadata.Annotations, fdbtypes.OutdatedConfigMapKey)
```

Comment:
Should we check if the Annotations contain that key and only then trigger an update?

Reply:
We're always going to be doing the update by the time we reach this branch, because we know the value for `LastConfigMapKey`, so I don't think checking if it already contains the `OutdatedConfigMapKey` is necessary.

Reply:
Right 👍